src/node_dependencies/traverse.js

```diff
@@ -53,8 +54,9 @@ const getDependencyNamesAndPathsForDependencies = async function ({
       }),
     ),
   )
-  const moduleNames = new Set(dependencies.flatMap((dependency) => [...dependency.moduleNames]))
-  const paths = new Set(dependencies.flatMap((dependency) => [...dependency.paths]))
+  const filteredDependencies = dependencies.filter(Boolean)
```

Comment:
When do we get a falsy value for a `dependency`?

Reply:
Oops. I redid this part a couple of times with different approaches and this `filter` call was an accidental leftover. Thanks for flagging!

Reply:
I'm going to remove this in the next PR, if that's okay.